src/examples/custom_reports.yml

```diff
@@ -0,0 +1,15 @@
+description: >
+  Publish custom results and reports.
+usage:
+  version: 2.1
+
+  orbs:
+    gradle: entur/gradle-orb@x.y
+
+  workflows:
+    test-with-custom-reports:
+      jobs:
+        - gradle/test:
```

Comment:
Perhaps we should use values that differs from default in the example?

Reply:
Will revisit this in next version, use of these parameters + `app_src_directory` is not 100%